addon/mixins/route.js

```diff
@@ -199,6 +200,17 @@ export default Ember.Mixin.create({
   },
 
   /**
+   Call additional functions after finding the infinityModel in the Ember data store.
+
+   @method afterInfinityModel
+   @param {Function} infinityModelPromise The resolved result of the Ember store find method. Passed in automatically.
+   @return {Ember.RSVP.Promise}
+   */
+  afterInfinityModel(infinityModelPromiseResult) {
+    return infinityModelPromiseResult;
```

Comment:
I'm not too sure if a call to `_super` is needed. You're right about it expecting `afterInfinityModel` to return the objects, which I don't like too much either.

What's your take on this? @kxcrl